recipe/meta.yaml

```diff
@@ -37,125 +41,96 @@ requirements:
     - coreutils  # [osx]
     - patchelf  # [linux]
     - unzip
-    - nodejs
+    - nodejs >=18
```

Comment:
Are we sure this is a good idea? nodejs has a per-major run-export, so building per version sounds much safer to me in terms of co-installability with other nodejs-using packages (that said, I'm not sure if https://github.com/conda-forge/rstudio-feedstock/pull/31/commits/96d47f1f668e6ee80698afb27e2f10a3e499faff was just intended as a debug commit to reduce the number of jobs)

Reply:
Nope. Then we should remove the run exports!

Reply:
I mean - ignore the run exports here. We only need nodejs for building, right?

Reply:
in fact, the run exports is probably `weak` so it shouldn't propagate to the run deps.

Reply:
True, if it's a build-only dep it shouldn't matter. 👍